docs/build.md

```diff
@@ -46,3 +53,5 @@ You can optionally compile each directly:
 make auraed      # compile and install auraed with cargo
```

Comment:
we should remove these instructions too if you're deleting the Makefiles

Reply:
Those commands are still valid. I was referring to the `auraed/Makefile` and `auraescript/Makefile`. I think they are leftover from the sub-repo days.

Reply:
`auraed/README.md` and `auraescript/README.md` still have some references to their own Makefiles. I can make a PR to clean them up.

Reply:
That would be welcome. Thank you